x-pack/docs/en/rest-api/security/get-api-keys.asciidoc

```diff
@@ -94,11 +104,80 @@ You can use the following example to retrieve the API key by ID:
 
 [source,console]
 --------------------------------------------------
-GET /_security/api_key?id=VuaCfGcBCdbkQm-e5aOx
+GET /_security/api_key?id=VuaCfGcBCdbkQm-e5aOx&with_limited_by=true
 --------------------------------------------------
 // TEST[s/VuaCfGcBCdbkQm-e5aOx/$body.id/]
 // TEST[continued]
 
+A successful call returns a JSON structure that contains the information of the API key:
+
+[source,js]
+--------------------------------------------------
+{
+  "api_keys": [ <1>
+    {
+      "id": "VuaCfGcBCdbkQm-e5aOx", <2>
+      "name": "my-api-key", <3>
+      "creation": 1548550550158, <4>
+      "expiration": 1548551550158, <5>
+      "invalidated": false, <6>
+      "username": "myuser", <7>
+      "realm": "native1", <8>
+      "metadata": { <9>
+        "application": "myapp"
+      },
+      "role_descriptors": { }, <10>
+      "limited_by": [  <11>
+        {
+          "role-power-user": {
+            "cluster": [
+              "monitor"
+            ],
+            "indices": [
+              {
+                "names": [
+                  "*"
+                ],
+                "privileges": [
+                  "read"
+                ],
+                "allow_restricted_indices": false
+              }
+            ],
+            "applications": [ ],
+            "run_as": [ ],
+            "metadata": { },
+            "transient_metadata": {
+              "enabled": true
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+--------------------------------------------------
+// NOTCONSOLE
+<1> The list of API keys that were retrieved for this request.
```

Comment:
Nit preference but I would omit notes on the obvious stuff, like `id` and `name` -- it mostly just clutters the doc and it's harder to find the important info IMO.

Reply:
Ah I see, it's a copy from below. Good with keeping as is, in that case.